devito/archinfo.py

```diff
@@ -107,15 +119,16 @@ def get_cpu_brand():
         # Fallback 1: it should now be fine to use psutil
         physical = psutil.cpu_count(logical=False)
         if not physical:
-            # Fallback 2: we might end up here on more exotic platforms such a Power8
-            # Hopefully we can rely on `lscpu`
+            # Fallback 2: we might end up here
```

Comment:
can we avoid breaking this line into three?

also, typo: 'such a' -> 'such as'

should we now drop the "or due to erroneous autodetection" part?

Reply:
Yes